spec/models/concerns/test_track/remote_model_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe TestTrack::Identity do
```

Comment:
Is this supposed to be describing something else?

Reply:
yes, sorry about that. will pr a fix